Assets/Code/Scripts/Loot.cs

```diff
@@ -0,0 +1,18 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+[CreateAssetMenu]
+
+public class Loot : ScriptableObject
+{
+    public Sprite lootSprite;
+    public string lootName;
+    public int dropChance;
```

Comment:
Może lepiej tutaj polecieć floatem i po prostu na procentach 0-1 to 0-100%

Reply:
Zmienione na float